cmd/helios-cli/cli_smoke_test.go

```diff
@@ -72,3 +72,17 @@ func TestCLI_Help(t *testing.T) {
 	}
 	_ = os.Stdout
 }
+
+func TestCLI_Version(t *testing.T) {
```

Comment:
gofmt: this file also shows formatting diffs; please run `gofmt -s -w cmd/helios-cli/*.go`.